deployment/prometheus-selfhosted.yml

```
@@ -6,11 +6,12 @@ global:
-
+  - job_name: 'vespa-data-ingest'
     metrics_path: /prometheus/v1/values
-
-
     static_configs:
       - targets: ['vespa:8080']
+      
+  - job_name: 'google_drive_ingestion'
+    metrics_path: /metrics  # default path in your Hono app
+    static_configs:
+      - targets: ['192.168.0.101:3000']
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)